### PR TITLE
#166240981 configure sequelize-cli with es6

### PR DESCRIPTION
**Pull request by deschantkn**
#### What does this PR do?
- Add config.js
- Configure migrations
- Configure es6 with sequelize cli

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#166240981

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,9 @@
...
 PORT=
-DATABASE_URL=
\ No newline at end of file
+DATABASE_URL=
+LOCAL_DB_USER=
+LOCAL_DB_PASSWORD=
+LOCAL_DB_NAME=
+TEST_DB_USER=
+TEST_DB_PASSWORD=
+TEST_DB_NAME=
```
Comment (Diama1) [inline .env.example]: It would be better if you put a new line

**Review by mifeille — APPROVED** [review]

**Review by EmyRukundo — APPROVED** [review]

**Review by Diama1 — APPROVED** [review]

**Review by rukundoeric — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (Quantum-35) [inline .env.example:9]: Could consider using a  database `connection URI` instead of saving a lot of DB credentials in `.env`

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require('@babel/register');
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('./src/config', 'config.js'),
+  'models-path': path.resolve('./src/models'),
+  'seeders-path': path.resolve('./src/seeders'),
+  'migrations-path': path.resolve('./src/migrations')
+}
```
Comment (Quantum-35) [inline .sequelizerc:10]: Move sequelize folders into one folder. You can name it `Sequelize`.For instance `src/sequelize/models`

**Review by Quantum-35 — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (deschantkn) [reply .env.example:9]: Okay I am on it

**Review by Quantum-35 — APPROVED** [review]

**Pull request merged**